faq/dedibox-in-scaleway-console.mdx

```diff
@@ -114,7 +114,7 @@ Yes, the API system is not impacted by account linking.
 
 
 ## What are the terms of the voucher offered for the migration?
-For the first customers following the linking process, a discount of 100€ is applied directly to their Scaleway Elements account. This discount is valid for six months after the link of your accounts. It is available on our Scaleway Elements cloud services ordered through the Scaleway console, excluding DDX and Elastic Metal products.
+For the first customers following the linking process, a discount of 100€ is applied directly to their Scaleway account. This discount is valid for six months after the link of your accounts. It is available on our Scaleway cloud services ordered through the Scaleway console, excluding Dedibox and Elastic Metal products.
```

Comment:
The first customers to follow the linking process will be granted a discount of 100€. This discount is directly applied to the Scaleway account and is valid for six months following the linking process. It is available on Scaleway cloud services ordered through the Scaleway console and does not include Dedibox and Elastic Metal products.

Reply:
```suggestion
The first customers to follow the linking process will be granted a discount of 100€. This discount is directly applied to the Scaleway account and is valid for six months following the linking process. It is available on Scaleway cloud services ordered through the Scaleway console and does not include Dedibox and Elastic Metal products.
```